src/dotnet/Program.cs

```diff
@@ -81,6 +81,7 @@ internal static int ProcessArgs(string[] args, ITelemetry telemetryClient = null
             var lastArg = 0;
             var cliFallbackFolderPathCalculator = new CliFallbackFolderPathCalculator();
             using (INuGetCacheSentinel nugetCacheSentinel = new NuGetCacheSentinel(cliFallbackFolderPathCalculator))
+            using (IFirstTimeUseNoticeSentinel firstTimeUseNoticeSentinel = new FirstTimeUseNoticeSentinel(cliFallbackFolderPathCalculator))
```

Comment:
That makes sense. But I can make this change when I switch the fallback folder to the dotnet hive.